<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>530</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>530</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>9</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(150, 150, 150);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QFrame" name="frame_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>120</width>
            <height>491</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(80, 80, 80);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QPushButton" name="pushButtonVenVisualizar">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>50</y>
             <width>91</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(85, 255, 255);</string>
           </property>
           <property name="text">
            <string>Visualuzar</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButtonSair">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>450</y>
             <width>121</width>
             <height>41</height>
            </rect>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 92, 70);</string>
           </property>
           <property name="text">
            <string>Sair</string>
           </property>
          </widget>
          <widget class="QFrame" name="frame_5">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>230</y>
             <width>120</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(27, 27, 27);
</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>111</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>13</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 85, 255);</string>
            </property>
            <property name="text">
             <string>     Cliente</string>
            </property>
           </widget>
          </widget>
          <widget class="QFrame" name="frame_6">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>120</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(27, 27, 27);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>2</y>
              <width>111</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>13</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 85, 255);</string>
            </property>
            <property name="text">
             <string>  Vendedor</string>
            </property>
           </widget>
          </widget>
          <widget class="QPushButton" name="pushButtonVenCriar">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>90</y>
             <width>91</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(85, 255, 255);</string>
           </property>
           <property name="text">
            <string>Criar</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButtonVenEditar">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>130</y>
             <width>91</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(85, 255, 255);</string>
           </property>
           <property name="text">
            <string>Editar</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButtonClienVisualizar">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>280</y>
             <width>91</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(85, 255, 255);</string>
           </property>
           <property name="text">
            <string>Visualuzar</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButtonClienCriar">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>320</y>
             <width>91</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(85, 255, 255);</string>
           </property>
           <property name="text">
            <string>Criar</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButtonClienEditar">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>360</y>
             <width>91</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(85, 255, 255);</string>
           </property>
           <property name="text">
            <string>Editar</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButtonVenExcluir">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>170</y>
             <width>91</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(85, 255, 255);</string>
           </property>
           <property name="text">
            <string>Excluir</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButtonClienExcluir">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>400</y>
             <width>91</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(85, 255, 255);</string>
           </property>
           <property name="text">
            <string>Excluir</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuVendedor">
    <property name="title">
     <string>Vendedor</string>
    </property>
    <addaction name="actionPequisar_Vendedor"/>
    <addaction name="actionTodos"/>
   </widget>
   <widget class="QMenu" name="menuCliente">
    <property name="title">
     <string>Cliente</string>
    </property>
    <addaction name="actionCliente_Especifico"/>
    <addaction name="actionto"/>
   </widget>
   <widget class="QMenu" name="menuOp_es">
    <property name="title">
     <string>Opções</string>
    </property>
    <addaction name="actionCriar_vendedor"/>
    <addaction name="actionEditar_Vendedor"/>
    <addaction name="actionCriar_Cliente"/>
    <addaction name="actionEditar_Vendedor_2"/>
    <addaction name="actionSair"/>
   </widget>
   <addaction name="menuVendedor"/>
   <addaction name="menuCliente"/>
   <addaction name="menuOp_es"/>
  </widget>
  <action name="actionPequisar_Vendedor">
   <property name="text">
    <string>Pequisar Vendedor</string>
   </property>
  </action>
  <action name="actionTodos">
   <property name="text">
    <string>Todos</string>
   </property>
  </action>
  <action name="actionCliente_Especifico">
   <property name="text">
    <string>Cliente Especifico</string>
   </property>
  </action>
  <action name="actionto">
   <property name="text">
    <string>Todos</string>
   </property>
  </action>
  <action name="actionCriar_vendedor">
   <property name="text">
    <string>Criar vendedor</string>
   </property>
  </action>
  <action name="actionEditar_Vendedor">
   <property name="text">
    <string>Editar Vendedor</string>
   </property>
  </action>
  <action name="actionCriar_Cliente">
   <property name="text">
    <string>Criar Cliente</string>
   </property>
  </action>
  <action name="actionEditar_Vendedor_2">
   <property name="text">
    <string>Editar Vendedor</string>
   </property>
  </action>
  <action name="actionVendedor">
   <property name="text">
    <string>Vendedor</string>
   </property>
  </action>
  <action name="actionCliente">
   <property name="text">
    <string>Cliente</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
